index.js

```diff
@@ -14,7 +14,7 @@ async function main() {
   const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
 
   // 1. Get the current version
-  const {version: newVersion} = __non_webpack_require__(pathJoin(core.getInput('path'), 'package.json'));
+  const {version: newVersion} = core.getInput('tag_prefix') + __non_webpack_require__(pathJoin(core.getInput('path'), 'package.json'));
```

Comment:
```suggestion
  const {version: newVersion} = core.getInput('tag_prefix') || "" + __non_webpack_require__(pathJoin(core.getInput('path'), 'package.json'));
```
to avoid having undefinedv2.2